README.md

```diff
@@ -58,20 +58,67 @@ pip install -U dataprep
+| Music | [Spotify](https://developer.spotify.com/)  | `OAuth2` |  [Spotify config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/spotify) |  [Spotify tutorial](https://sfu-db.github.io/dataprep/user_guide/connector/DC_Spotify_tut.html) |  Comprehensive API for retrieving albums, artists, and tracks metadata. |
+| Restaurants | [Yelp](https://www.yelp.com/developers)  | `Bearer Token` |  [Yelp config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/yelp) |  [Yelp Jupyter Notebook](https://github.com/sfu-db/dataprep/blob/develop/examples/DataConnector_Yelp.ipynb) |  Leading API to access restaurant information by location.  |
+| Science | [DBLP](https://dblp.org/faq/13501473.html) | No |  [DBLP config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/dblp) |  [DBLP Jupyter Notebook](https://github.com/sfu-db/dataprep/blob/develop/examples/DataConnector_DBLP.ipynb) |   Open bibliographic API for computer science publications.  |
+| Social Media | [Youtube](https://developers.google.com/youtube)  | `API Key` |  [Youtube config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/youtube) |  [Youtube Jupyter Notebook](https://github.com/sfu-db/dataprep/blob/develop/examples/DataConnector_Youtube.ipynb) |  API for retrieving Youtube's content information.  |
+| Finance | [Finnhub](https://finnhub.io/)  | `API Key` |  [Finnhub config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/finnhub) |  [Finnhub Jupyter Notebook](https://github.com/sfu-db/dataprep/blob/develop/examples/DataConnector_Finnhub.ipynb) |   Comprehensive API for financial, market, and economic data.  |
+| Music | [Musixmatch](https://developer.musixmatch.com/)  | `API Key` |  [Musixmatch config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/musixmatch) | Coming soon |  Leading API for searching music lyrics.  |
+| Weather | [OpenWeatherMap](https://openweathermap.org/api)  | `API Key` |  [OpenWeatherMap config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/openweathermap) | Coming soon |  API for retrieving current and historical weather data. |
+| Lifestyle | [Spoonacular](https://spoonacular.com/food-api)  | `API Key` |  [Spoonacular config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/spoonacular) | Coming soon |  Recipe, food, and nutritional information API.  |
+
+If you want to connect with a different web API, Connector is designed to be easy to extend. You just have to write a simple [configuration file](https://sfu-db.github.io/dataprep/user_guide/connector/connector.html) to support the new web API. This configuration file describes the API's main attributes like the URL, query parameters, authorization method, pagination properties, etc.  
```

Comment:
Should this link to the DataConnectorConfigs repo instead?

Reply:
Done.